### ISSUE TITLE
async endpoint doesn't appear to post result to callback URL on error

### ISSUE THREAD

**Issue by andrewsu** [biothings/biothings_explorer#349]
In this query result https://arax.ncats.io/?r=ef6d4f87-f53c-4f8c-9aba-a804d74c5761  BTE shows an eternally "running status"

![image](https://user-images.githubusercontent.com/2635409/140543307-a223dd6d-1ecc-42c8-ae6a-3e95f9faabfc.png)

but loading the BTE-specific primary key shows that a query status URL is provided https://arax.ncats.io/?r=88a768fa-474c-40fd-af7d-5789e3587035

![image](https://user-images.githubusercontent.com/2635409/140543542-74877d0c-e8ef-449d-8725-8c8cceca119f.png)

Checking that query status link shows that the job completed with an error http://api.bte.ncats.io/v1/check_query_status/feP9Tf8F7Y

![image](https://user-images.githubusercontent.com/2635409/140543709-794fe754-40ae-472c-8625-4cc8e18ba8c9.png)

So in this case it appears that BTE did not post an error message back to the ARS so that the status at the first link above can be updated.  (As far as I recall, when BTE successfully executes the query, posting the result to the callback URL works as intended.)

**Comment by colleenXu** [comment]
Looks like the query was something like this: 

```
  "message": {
    "query_graph": {
      "edges": {
        "e00": {
          "object": "n01",
          "predicates": [
            "biolink:related_to"
          ],
          "subject": "n00"
        },
        "e01": {
          "object": "n02",
          "predicates": [
            "biolink:related_to"
          ],
          "subject": "n01"
        }
      },
      "nodes": {
        "n00": {
          "categories": [
            "biolink:Disease"
          ],
          "ids": [
            "MONDO:0008753"
          ]
        },
        "n01": {
          "categories": [
            "biolink:NamedThing"
          ]
        },
        "n02": {
          "categories": [
            "biolink:Gene"
          ],
          "ids": [
            "NCBIGene:3081"
          ]
        }
      }
    }
  },
```

**Comment by marcodarko** [comment]
A request wasn't posted when an error occurred due to return statement terminating the method, addressed in this pull request: https://github.com/biothings/BioThings_Explorer_TRAPI/pull/358

**Comment by tokebe** [comment]
Fixed by #358
